### ISSUE TITLE
Push fails in github action but succeeds locally

### ISSUE THREAD

**Issue by vegar** [10gen/realm-cli#158]
```
❯ realm-cli -v
realm-cli version 2.0.0-beta.6
```

I've made myself a github action that first exports the app, copies new files into /hosting/files and then tries to push the changes using the command line :

`realm-cli push --local realm --remote *** --include-hosting --reset-cdn-cache -y`

When running this command locally on a windows 10 machine, I'm rewarded with the output

```
Determining changes
Import hosting assets
Reset CDN cache
Successfully pushed app up: ***
```

The github action fails though with the following output:

```
  shell: /usr/bin/bash -e {0}
Determining changes
push failed: EOF
Error: Process completed with exit code 1.
```

What can be the reason for this?
I've checked that the github action copies the new files into the /hosting/files folder before pushing.

**Comment by stoefln** [comment]
Same here on my local Mac M1

**Comment by benoitwerner** [comment]
I have the same problem, where you able to resolve it?

**Comment by stoefln** [comment]
No. We switched away from realm. We were facing too many issues.

**Comment by benoitwerner** [comment]
> No. We switched away from realm. We were facing too many issues.

Thanks for your reply!
